src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,666 @@
+                          assertEquals("FOO", new Foo().getMsg());
+                          assertEquals("FOOecho", new Foo().getMsg("echo"));
+                          assertEquals("BAR", new Bar().getMsg());
+                          assertEquals("BARecho", new Bar().getMsg("echo"));
+                      }
+                  }
+              }
+              """)
+        );
+    }
+
+    @Test
+    void mockUpInnerStatementTest() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import mockit.Mock;
+              import mockit.MockUp;
+              
+              import org.junit.Test;
+              import static org.junit.Assert.assertEquals;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      new MockUp<MyClazz>() {
+                          final String msg = "newMsg";
+              
+                          @Mock
+                          public String getMsg() {
+                              return msg;
+                          }
+                      };
+              
+                      // Should ignore the newClass statement
+                      new Runnable() {
+                          @Override
+                          public void run() {
+                              System.out.println("run");
+                          }
+                      };
+                      assertEquals("newMsg", new MyClazz().getMsg());
+                  }
+              
+                  public static class MyClazz {
+                      public String getMsg() {
+                          return "msg";
+                      }
+                  }
+              }
+              """, """
+              import org.junit.Test;
+              import org.mockito.MockedConstruction;
+              
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.AdditionalAnswers.delegatesTo;
+              import static org.mockito.Answers.CALLS_REAL_METHODS;
+              import static org.mockito.Mockito.*;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      final String msg = "newMsg";
+                      MockUpTest.MyClazz mockObjMockUpTestMyClazz = mock(MockUpTest.MyClazz.class, withSettings().defaultAnswer(CALLS_REAL_METHODS));
+                      doAnswer(invocation -> {
+                          return msg;
+                      }).when(mockObjMockUpTestMyClazz).getMsg();
+                      try (MockedConstruction mockConsMockUpTestMyClazz = mockConstructionWithAnswer(MockUpTest.MyClazz.class, delegatesTo(mockObjMockUpTestMyClazz))) {
```

Comment:
I liked your previous variable naming convention of of mockConsMyClass, I think leave it as such ?